category_theory/limits/cones.lean

```diff
@@ -39,12 +39,34 @@ lemma cones_obj (X : C) : F.cones.obj X = ((const J).obj X ⟹ F) := rfl
 natural transformations from `F` to the constant functor with value `X`.
 An object corepresenting this functor is a colimit of `F`.
 -/
-def cocones : C ⥤ Type _ := (const J) ⋙ (coyoneda.obj F)
+def cocones : C ⥤ Type v := (const J) ⋙ (coyoneda.obj F)
 
 lemma cocones_obj (X : C) : F.cocones.obj X = (F ⟹ (const J).obj X) := rfl
 
 end functor
 
+section functoriality
```

Comment:
I guess we have a lack of different names for different trivial concepts, but we already have an unrelated thing called `cone.functoriality` below... Maybe that one should be called something like `map` or `whisker_right`?

Reply:
I fixed this by making the section anonymous.